Xamarin.Forms.Platform.iOS/Renderers/PageRenderer.cs

```diff
@@ -215,6 +223,22 @@ public override UIKit.UIStatusBarAnimation PreferredStatusBarUpdateAnimation
 			}
 		}
 
+		void UpdateUseSafeArea()
+		{
+			if (!Forms.IsiOS11OrNewer) return;
+
+			if (!UsingSafeArea)
+			{
+				var safeAreaInsets = SafeAreaInsets;
+				if (safeAreaInsets == Page.Padding)
+					Page.Padding = default(Thickness);
```

Comment:
setting the `Padding` property from the renderer is even worse than the original version

Reply:
Why? This is being triggered only when that property is changed so it's functionally identical. 

Plus it's being set from inside the context of iOS so it has more accurate platform context. When it was being set inside shared code then we had to add if statements to make sure it wasn't effecting other platforms.  Which seems more inelegant to me.

Reply:
@StephaneDelcroix Do you agree with @PureWeen? Let us know how to proceed! Thanks!